### ISSUE TITLE
Doubt about the construction of WSI representation

### ISSUE THREAD

**Issue by lsnls** [mahmoodlab/PANTHER#28]
Hi, authors!

Thank you for your valuable work! As shown in formula (1), the mapping function is based on the similarity between the patch embedding and the prototype. Then, in formula (6), the sum of the mapping function is represented using three estimated parameters. Could you please explain why these three parameters can sufficiently represent the mapping function?
![image](https://github.com/user-attachments/assets/2461209f-c81c-4443-8b28-e6ceee439321)
![image](https://github.com/user-attachments/assets/9177e147-8929-42d2-bd6d-6b5e1d1f2957)

Looking forward to your reply! Thank you so much!

Best regards,
Suoni

**Comment by andrewsong90** [comment]
Hi @lsnls 

Thanks for your interest in the work.
To clarify, Eq. 1 is meant to be very generic formulation - As long as the function \phi_c takes in the patch embeddings and  prototype c, it can be any function.

Eq. 6 is my take on what these functions could actually look like. Borrowing ideas from Gaussian Mixture models, I parameterized \phi with parameters of GMM. If we _assume_ that each patch embedding is generated from GMM distribution, then the estimated parameters should be sufficient summary statistics for the distribution.

Of course, you can argue that GMM might not be the "right" modeling approach - In that case \phi can be more complicated distributions/functions of your choice and the parameterization would be different.

Long story short, Eq.1 is general description of the problem and Eq.6 is dependent on the modeling choice.

Hope this helps

**Comment by lsnls** [comment]
Hi, @andrewsong90 

I understand. Thanks a lot for your quick response!
